src/elf.rs

```diff
@@ -475,8 +475,10 @@ impl<'a> Elf<'a> {
             }
         }
     }
-    pub fn add_definition(&mut self, name: &str, data: &'a [u8], decl: &artifact::DefinedDecl) {
-        let def_size = data.len();
+    pub fn add_definition(&mut self, def: artifact::Definition<'a>) {
```

Comment:
Why was this breaking change added/needed?

Reply:
I don't think this is a breaking change as the `elf` mod is not public and nothing in it is re-exported. This change is necessary when the elf support gets implemented, as only the `Definition` contains the `symbols`.

Reply:
Ah yup sorry, missed we were in the elf module :)